README.md

```diff
@@ -57,6 +57,18 @@ Apply `slo: "true"` under labels section of any alerting rule.
 ##### Note
 SLO alerts should be labeled with `severity: critical`
 
+### Labeling Control Plane alerts
+
```

Comment:
> routed further more to AppSRE
Not clear enough, IMO

Reply:
I rephrased it, I hope it is clearer now